src/main/resources/application-local.properties

```diff
@@ -21,3 +21,7 @@ aws.sns.audit-search.topic-name=hocs-audit-topic
 aws.sns.audit-search.arn=arn:aws:sns:${aws.sns.config.region}:${aws.sns.audit-search.account.id}:${aws.sns.audit-search.topic-name}
 
 spring.datasource.hikari.minimumIdle=2
+
+migration.userid=f55d4370-3955-44b8-91d0-f1b08c967a2b
```

Comment:
Should this have been committed?

Reply:
It was randomly generated. I will remove it.